dev-tools/mage/linter.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	linterVersion    = "v1.55.2"
+	linterVersion    = "v2.0.2"
```

Comment:
Do you have to update `.github/workflows/golangci-lint.yml` as well?

Reply:
Yes! Thanks for catching this :D.